generic3g/tests/Test_ConvertUnitsTransform.pf

```diff
@@ -1,26 +1,43 @@
 #include "MAPL_TestErr.h"
 #include "unused_dummy.H"
 
+#if defined(_FAILURE)
+#  undef _FAILURE
+#endif
+#define _FAILURE _SUCCESS-1
+
+#if defined(_RETURN)
+#  undef _RETURN
+#endif
+#define _RETURN(A) if(present(rc)) rc=A; return
+
```

Comment:
Most of these are all in the test macros in  ./include/MAPL_TestErr.h

_FAILURE is missing, so just add it there.